Formula/s/sile.rb

```diff
@@ -1,18 +1,13 @@
 class Sile < Formula
   desc "Modern typesetting system inspired by TeX"
   homepage "https://sile-typesetter.org"
-  license "MIT"
-  revision 1
 
-  # TODO: With 0.15 release:
-  # - Remove `cosmo` resource and corresponding references in install
-  # - Switch `lua` dependency to `luajit` and clean up `lua` references
-  stable do
-    url "https://github.com/sile-typesetter/sile/releases/download/v0.14.17/sile-0.14.17.tar.xz"
-    sha256 "7f89bedecedabb5168250ad9dd80c09ed289c8e88c3d0d756d2d1d92ee065e04"
+  url "https://github.com/sile-typesetter/sile/releases/download/v0.15.3/sile-0.15.3.tar.zst"
+  sha256 "88d24a70710e301ba3400351a82c5f28b49cebcd637b7a59d28b001a56b398c6"
 
-    depends_on "lua"
-  end
+  url "https://github.com/sile-typesetter/sile.git", branch: "develop" if build.head?
```

Comment:
```suggestion
  url "https://github.com/sile-typesetter/sile.git", branch: "develop"
```

Reply:
This suggestion isn't correct, the `--HEAD` build has a different URl than the default. But we *could* drop the different branch bit, that is kind of obsolete now.